app/controllers/courses_controller.rb

```diff
@@ -53,4 +63,33 @@ def _create_response_payload_schema
     }
   end
 
+  def _fetch_metadatas_response_payload_schema
+    {
+      '$schema': JSON_SCHEMA,
+
+      'type': 'object',
+      'properties': {
+        'course_responses': {
+          'type': 'array',
+          'items': {'$ref': '#definitions/course_metadata'},
+          'minItems': 0,
+          'maxItems': 1000,
+        },
+      },
+      'required': ['course_responses'],
+      'additionalProperties': false,
+      'standard_definitions': _standard_definitions,
+      'definitions': {
+        'course_metadata': {
+          'type': 'object',
+          'properties': {
+            'uuid': {'$ref': '#standard_definitions/uuid'}
```

Comment:
Might wanna return the ecosystem uuid here (maybe call it `initial_ecosystem_uuid` to make it clear that we won't bother trying to find the latest ecosystem).

The reason for that is that we might want to filter what courses we care about based on the ecosystem.

Another useful key is `is_demo` but that would require changing the create_course schema to add that information.